src/utils/utils.ts

```diff
@@ -168,40 +174,14 @@ export const useQueryWithClient = <
 
 export const parseJsonErrorMessage = (error: string) => {
   try {
-    const parsedError = JSON.parse(error);
+    const parsedError = JSON.parse(error) as { status: string; message: string };
```

Comment:
We should verify the schema before type casting it.

Reply:
I’ve added the check `if (parsedError.status && parsedError.message)`